plan.go

```diff
@@ -2837,17 +2839,20 @@ func (r *emptyFieldsPlan) do(ctx *execCtx, f func(id interface{}, data []interfa
 
 type wrapFilterPlan struct {
 	*filterDefaultPlan
+	once  sync.Once
```

Comment:
sync.Once is probably an overkill. Instances of plans are not used concurrently, IIRC. And if they're used concurrently then we cannot have one state variable to properly guard the evaluation because the concurrent executions can have different contexts with diference arguments passed to the query etc.

I guess execCtx is the correct place where to keep the "evaluate once" guard. I believe execCtx is never shared among statement executions.